ci/docker-compose.ci.yml

```
@@ -0,0 +1,49 @@
+version: '3.4'
+
+services:
+  db:
+    image: postgres
```

```suggestion
    image: postgres:14
```

PG 15 has some breaking changes w/r to public schema, so easiest for now is to pin to 14